modules/tiles/src/tileset/helpers/bounding-volume.js

```diff
@@ -28,101 +55,30 @@ const scratchTransform = new Matrix4();
  * @param [result] The object onto which to store the result.
  * @returns The modified result parameter or a new TileBoundingVolume instance if none was provided.
  */
-export function createBoundingVolume(boundingVolumeHeader, transform, result) {
+export function transformBoundingVolume(boundingVolumeHeader, transform, result) {
   assert(boundingVolumeHeader, '3D Tile: boundingVolume must be defined');
 
+  const {box, region, sphere} = boundingVolumeHeader;
   // boundingVolume schema:
   // https://github.com/AnalyticalGraphicsInc/3d-tiles/blob/master/specification/schema/boundingVolume.schema.json
-  if (boundingVolumeHeader.box) {
-    return createBox(boundingVolumeHeader.box, transform, result);
-  }
-  if (boundingVolumeHeader.region) {
-    // [west, south, east, north, minimum height, maximum height]
-    // Latitudes and longitudes are in the WGS 84 datum as defined in EPSG 4979 and are in radians.
-    // Heights are in meters above (or below) the WGS 84 ellipsoid.
-    const [west, south, east, north, minHeight, maxHeight] = boundingVolumeHeader.region;
-
-    const northWest = Ellipsoid.WGS84.cartographicToCartesian(
-      [degrees(west), degrees(north), minHeight],
-      scratchNorthWest
-    );
-    const southEast = Ellipsoid.WGS84.cartographicToCartesian(
-      [degrees(east), degrees(south), maxHeight],
-      scratchSouthEast
-    );
-    const centerInCartesian = new Vector3().addVectors(northWest, southEast).multiplyScalar(0.5);
-    const radius = new Vector3().subVectors(northWest, southEast).len() / 2.0;
-
-    // TODO improve region boundingVolume
-    // for now, create a sphere as the boundingVolume instead of box
-    return createSphere(
-      [centerInCartesian[0], centerInCartesian[1], centerInCartesian[2], radius],
-      new Matrix4()
-    );
-  }
 
-  if (boundingVolumeHeader.sphere) {
-    return createSphere(boundingVolumeHeader.sphere, transform, result);
-  }
+  switch (true) {
+    case box != void 0:
+      return transformOrientedBoundingBox(box.clone(), transform);
+    //return box.clone().transform(transform);
 
-  throw new Error('3D Tile: boundingVolume must contain a sphere, region, or box');
-}
+    case region != void 0:
+      return transformBoundingSphere(region.clone(), new Matrix4());
```

Comment:
In the tests (`yarn bootstrap; yarn test browser`, `region` is not a math.gl object so it doesn't have the `clone` method. It is just an array. 

I am not sure that we have a region class.

Ot it could be that this object was parsed on a worker and serialized and deserialized on its way back to the main thread. 

@belom88 Can you assist?

```
test.only('Tileset3D#loads tiles in tileset', async t => {
``

Reply:
Alexander creates sphere from region as it was done before. Now this logic is moved to '3d-tiles' loader. I'm not sure but possibly he should rename property `region`->`sphere` because this naming is bit confusing.